Source/reactions/Castro_react.cpp

```diff
@@ -139,32 +139,56 @@ Castro::react_state(MultiFab& s, MultiFab& r, Real time, Real dt)
 
                 if (do_burn) {
 
-                    // Add burning rates to reactions MultiFab, but be
-                    // careful because the reactions and state MFs may
-                    // not have the same number of ghost cells.
+                    // update the state
 
-                    if (reactions.contains(i,j,k)) {
+                    if (U.contains(i,j,k)) {
```

Comment:
This contains check would no longer be necessary, since the box is defined with respect to the same array

Reply:
fixed